tests/test_dashboard.py

```diff
@@ -0,0 +1,109 @@
+import time
+from tests.conftest import logging
+
+logger = logging.getLogger(__name__)
+
+def test_get_metadata_dashboard(api_session, base_url):
+    session = api_session
+    base_url = base_url
+    url = base_url + "/api/v2/dashboards"
+    data = {
+        "cfxql_query":"*",
```

Comment:
All these are default parameters. You should test with default parameters by not passing them?